test/server/cards/02_SHD/events/TimelyIntervention.spec.ts

```diff
@@ -0,0 +1,37 @@
+describe('Timely Intervention', function () {
+    integration(function (contextRef) {
+        describe('Timely Intervention\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        leader: 'hera-syndulla#spectre-two',
+                        hand: ['timely-intervention', 'rebel-pathfinder', 'alliance-xwing'],
+                        groundArena: ['battlefield-marine'],
+                        resources: ['tech#source-of-insight', 'atst', 'atst', 'atst']
+                    },
+                    player2: {
+                        groundArena: ['isb-agent'],
+                    }
+                });
+            });
+
+            it('should allow the player to play a unit from his hand, paying its cost, and give it ambush', function () {
```

Comment:
Could we also have a test where we smuggle it just to confirm that works? I don't feel the need to do that with every smuggle card but the timing is a little tricky on this one so I just want some additional confirmation that it works right